docs/operation.md

```diff
@@ -7,8 +7,7 @@ This document describes how to start your epoch node installed using a release b
 The instructions below assume that:
 * The node is deployed in directory `/tmp/node`;
 * No custom peers are specified under the `peers:` key in the config. If the `peers:` key is undefined, the *testnet* seed peers (built-in in the package source) are used.
-* The external HTTP endpoint of the user API of the node can be contacted at 127.0.0.1 port 3003;
-* The internal HTTP endpoint of the user API of the node can be contacted at 127.0.0.1 port 3103.
```

Comment:
I see the internal API is not used in this docs, thus the removal of the line?

Reply:
Yes.